<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Enroll</class>
 <widget class="QWidget" name="Enroll">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/assets/assets/How-Does-the-Attendance-Management-System-Help-the-Employees (1).jpg);
color: black;
</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>371</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="styleSheet">
       <string notr="true">color: purple;
font-size: 20px;
font: Arial;</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameInp">
      <property name="styleSheet">
       <string notr="true">height: 25px;
color: black;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="emailLabel">
      <property name="styleSheet">
       <string notr="true">color: purple;
font-size: 20px;
font: Arial;</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="emailInp">
      <property name="styleSheet">
       <string notr="true">height: 25px;
color: black;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="addressLabel">
      <property name="styleSheet">
       <string notr="true">color: purple;
font-size: 20px;
font: Arial;</string>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="addressInp">
      <property name="styleSheet">
       <string notr="true">height: 25px;
color: black;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="roleLabel">
      <property name="styleSheet">
       <string notr="true">color: purple;
font-size: 20px;
font: Arial;</string>
      </property>
      <property name="text">
       <string>Role</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="roleInp">
      <property name="styleSheet">
       <string notr="true">height: 25px;
color: black;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">opacity: 1;
 padding: 10px;
 box-shadow: 2px 2px grey;
 font-size: 18px;
 color: white;
 background: green; 
border-radius: 5px;
z-index: 1;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: green;
 font-size:30px;
opacity: 0.8;</string>
   </property>
   <property name="text">
    <string>Enroll New Staff</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">opacity: 1;
 padding: 10px;
 box-shadow: 2px 2px grey;
 font-size: 18px;
 color: white;
 background: red; 
border-radius: 5px;
z-index: 1;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
